### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by BenMKT**
-Added basic CSS styling for my Portfolio project desktop version as per figma design.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -386,3 +372,516 @@ button[type]:hover {
...
+@media screen and (min-width: 375px) {
+  nav {
+    display: none;
+  }
+
+  .top {
+    background-image: url(img/header-shapes\ mobile.png);
+    background-size: cover;
+    background-repeat: no-repeat;
+  }
```
Comment (IngDavidTM) [inline styles.css:385]: - [ ] You have done an amazing job with the styling of your project! :+1: However, to ensure consistency with the Figma template, it is necessary to have a rounded bottom left corner on the main page. Therefore, kindly make this adjustment to match the project requirements and maintain design consistency.

**Your design**

Screenshot from 2023-04-07 19-27-38

**Figma**

Screenshot from 2023-04-07 19-28-05

File: styles.css [hunk 2]
```diff
@@ -386,3 +372,516 @@ button[type]:hover {
...
+    display: flex;
+    flex-direction: column;
+    background-color: #6070ff;
+    background-image: url(img/contact\ form\ background\ shapes@2x.png);
+    border-top-left-radius: 5em;
+    background-size: contain;
+    background-repeat: no-repeat;
+    margin: 0;
+    border-style: none;
+  }
```
Comment (IngDavidTM) [inline styles.css:836]: - [ ] Congratulations on the excellent styling of your project's contact form! :+1: However, to ensure consistency with the Figma template's layout, I suggest centering the form in the middle of the page. By doing so, your project's contact form will match the design and layout specifications required by the project. Please make the necessary adjustments accordingly.

**Your design**

Screenshot from 2023-04-07 19-26-11

**Figma**

Screenshot from 2023-04-07 19-29-30

File: styles.css [hunk 3]
```diff
@@ -386,3 +372,516 @@ button[type]:hover {
...
+    font-size: 1.25em;
+    font-weight: 500;
+    display: flex;
+    margin-top: 40px;
+    flex: 0;
+  }
+
+  .last .list {
+    text-align: left;
+  }
```
Comment (IngDavidTM) [inline styles.css:753]: - [ ] Great job with your project so far! :+1: However, it seems like there is a lot of space between your About Me and Contact Me sections. Please double-check the spacing and make any necessary adjustments to ensure that the space between the two sections is consistent with the project's requirements. Maintaining the correct spacing will help ensure that your project meets the design and layout specifications.

**Your design**

Screenshot from 2023-04-07 19-37-40

**Figma**

Screenshot from 2023-04-07 19-37-06

**Review by IngDavidTM — CHANGES_REQUESTED** [review]
## Hi @BenMKT  🙋‍♂️, 

Good job so far!
You have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!



## Top highlight 🎯
- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 

## Suggested changes

- [x] _Check the comments under the review._ ⬇

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 1 above)
Comment (BenMKT) [reply styles.css:385]: -Added `border-bottom-left-radius` to the `.top` class to conform to Figma design and resolve identified bug.

File: styles.css (hunk 2 above)
Comment (BenMKT) [reply styles.css:836]: -Added center properties to the `fieldset` element in the contact section with slight modifications on individual `input` fields to resolve the observed bug above.

File: styles.css (hunk 3 above)
Comment (BenMKT) [reply styles.css:753]: -Corrected spacing between the about section and contact form section by reducing the `margin-top` property in the contact section to resolve identified bug above.

File: styles.css [hunk 4]
```diff
@@ -386,3 +372,523 @@ button[type]:hover {
...
+    background-color: #6070ff;
+    background-image: url(img/contact\ form\ background\ shapes@2x.png);
+    border-top-left-radius: 100px;
+    background-size: contain;
+    background-repeat: no-repeat;
+    margin: 0;
+    border-style: none;
+    align-items: center;
+    justify-content: center;
+  }
```
Comment (Shedrack-Sunday) [inline styles.css:843]: - [ ] You've done well making an attempt on the changes suggested by the previous reviewer in this section 👏 . However, the centre view as seen on the figma template needs to be adjusted. 
-  You'll notice the form is not really at the centre, as the left margin has higher width than the right margin. From your design, it seems the centring works for the section of the `contact me` and paragraph  but not the form input. Kindly check the class that holds them, and possibly wrap the form input inside to have same centre view.
- I've made some markings on your view to show you the delicate sections. These changes are needed to help you stick to  requirements, and have an almost 100% replica of the original design which will be same for clients or hiring managers in future 👍 

## Your view:
image

## :
image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @BenMKT         👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well implementing the requirements,  However, there are a few you need to work on, to move to the next project. 💯

 


## Highlight
✔️ Proper git-flow used
✔️ moderate UI compared to Figma template
  
# Required Changes ♻️

- [x] I must admit you have really put in the effort so far in this project, however, the changes required below need to be implemented :
- The little adjustment needed on this project has to do with styling, of which you've done almost everything, but just a little more touch left on the contact form

_check the comment section for a detailed review_ 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: styles.css (hunk 4 above)
Comment (BenMKT) [reply styles.css:842]: -Created a `form` styles selector, removed `margin` properties from the individual `input` elements, and inserted centering styles into the `form` selector to resolve the issue.

File: styles.css [hunk 5]
```diff
@@ -386,3 +372,528 @@ button[type]:hover {
...
+    flex: 0;
+  }
+
+  .last .list {
+    text-align: left;
+  }
+
+  .rule {
+    height: 1px;
+    width: 684px;
```
Comment (binhussen) [inline styles.css:761]: - [Optional] Kindly, don't give fixed width for your horizontal line,the fixed width will make to horizontal to be out of the parent element, which we do n't went to,please remove it we need the default(parent inherited) here to stick the Figma design :+1:

File: styles.css [hunk 6]
```diff
@@ -386,3 +372,528 @@ button[type]:hover {
...
+    flex-direction: row;
+  }
+
+  .container .main {
+    background: none;
+  }
+
+  .about .main {
+    display: flex;
+    flex-wrap: wrap;
```
Comment (binhussen) [inline styles.css:512]: - [Optional] Kindly, The flex-wrap CSS property sets whether flex items are forced onto one line or can wrap onto multiple lines. If wrapping is allowed, it sets the direction that lines are stacked, that we don't went in our case, so please remove it to keep the design from wrapping the `about section` on `contact section` :+1:

**Review by binhussen — APPROVED** [review]
Hi @BenMKT , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️
## Optional suggestions
- [Optional] Kindly , you missed some part of the [readme](https://github.com/microverseinc/readme-template), please be sure that the only optional section is `FAQ` so Update it to proffessional repository :+1: 
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 6 above)
Comment (BenMKT) [reply styles.css:512]: -Removed the `flex-wrap` property to avoid this bug.

File: styles.css (hunk 5 above)
Comment (BenMKT) [reply styles.css]: -Replaced all fixed units with relative `em` and `rem` units for more responsiveness.

**Comment by BenMKT** [comment]
-Final responsive desktop version with `@media queries` included with relevant breakpoints for mobile.

**Pull request merged**